torchgeo/datasets/eurocrops.py

```diff
@@ -0,0 +1,285 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""EuroCrops dataset."""
+
+import csv
```

Comment:
We also have a pandas dependency if you find that easier to work with, but csv is also fine for simple things.